cpp/arcticdb/entity/types.hpp

```diff
@@ -115,7 +115,10 @@ enum class ValueType : uint8_t {
 
     UTF_DYNAMIC = 11,
     ASCII_DYNAMIC = 12,
-    /// Used to represent null types. Each type can be converted to Empty and Empty can be converted to each type.
+    // EMPTY is used by empty columns whose type has not been explicitly specified.
```

Comment:
It's used not only for empty columns, but for columns full of `None` values as well. E.g. `pd.DataFrame({"c": [None, None, None]})` would create a column of EMPTY type in Arctic

Reply:
Done with ff99224.